server/src/test/integration/org/apache/cassandra/sidecar/routes/tokenrange/BaseTokenRangeIntegrationTest.java

```diff
@@ -305,4 +332,20 @@ private void validateRanges(List<TokenRangeReplicasResponse.ReplicaInfo> replica
                                 .map(TokenRangeReplicasResponse.ReplicaInfo::end)
                                 .anyMatch(s -> s.equals("9223372036854775807"))).isTrue();
     }
+
+    protected @NotNull Set<String> getDcReplication(CassandraIntegrationTest annotation)
+    {
+        Set<String> dcReplication;
+        if (annotation.numDcs() > 1)
+        {
+            createTestKeyspace(ImmutableMap.of("replication_factor", DEFAULT_RF));
+            dcReplication = Sets.newHashSet(Arrays.asList("datacenter1", "datacenter2"));
```

Comment:
NIT: use JDK primitive types
```suggestion
            createTestKeyspace(Map.of("replication_factor", DEFAULT_RF));
            dcReplication = Set.of("datacenter1", "datacenter2");
```